js/app.jsx

```diff
@@ -861,8 +861,8 @@ React.render(
   <Routes>
     <Route handler={App}>
       <Route name="app" path="/" handler={BigData} ignoreScrollBehavior={true} />
-      <Route name="job" path="/job/:jobId" handler={Job} />
-      <Route name="logs" path="/job/:jobId/logs" handler={JobLogs} />
+      <Route name="job" path="/job/:jobId/?" handler={Job} />
+      <Route name="logs" path="/job/:jobId/logs/?" handler={JobLogs} />
```

Comment:
Let me look into this, I saw a bug that was filed about this if they've fixed it we may just be able to update the library version